chia/rpc/wallet_rpc_api.py

```diff
@@ -1441,12 +1447,17 @@ async def nft_get_info(self, request: Dict) -> Optional[Dict]:
     async def nft_add_uri(self, request) -> Dict:
         assert self.service.wallet_state_manager is not None
         wallet_id = uint32(request["wallet_id"])
-        uri = request["uri"]
+        # Note metadata updater can only add one uri for one field at once
```

Comment:
So this means that if I try to add a meta_uri and license_uri in the same call, the update will fail?

If that's the case, we should raise a ValueError indicating that only a single URI can be added per call.

Reply:
Or is the limitation once per spend?

Reply:
You can add one uri for data-uri, metadata-uri and license-uri independently in spend, check the unit test.

Reply:
The "for one field at once" part of the comment confused me. Maybe the comment can clarify that. Something like:

# Note metadata updater can only add a single URI of each type per spend. Adding multiple URIs of the same type will require multiple spends.